kubernetes_manager/tests/test_cases.py

```diff
@@ -128,7 +128,11 @@ def test_model(self):
         self.assertIsNotNone(obj.pk)
 
     def test_get_obj(self):
```

Comment:
Should probably write a few tests... one that creates w/o volumes, one that creates with one volume and one that creates with 2 or more volume

Reply:
yes

Reply:
added